requirements/bigquery_extras.txt

```diff
@@ -1,5 +1,4 @@
-google-cloud-core~=1.4
+# TODO: Upgrade google-cloud-bigquery to ~2.0
 google-cloud-bigquery~=1.28
-google-auth>=1.20.1,<2.0dev
-google-apitools>=0.5.31,<0.6
-pandas~=1.1
\ No newline at end of file
+pandas>=1.1,<2
+six>=1.14
```

Comment:
Same case as with airflow

Reply:
It is *extremely* unlikely, that Six will recieve any major update ever given that Python 2.x is officially unsupported.